### PR TITLE
Janibuffs

### PR DESCRIPTION
**Pull request by benj8560**
## About The Pull Request

	Long begged for buffs to janitors. Some decent stat buffs 20 TGH and VIG. 
	Basic burrow sealing tools. Pickaxe and hammer.
	Pure cosmetic change switching their generic small pouch over to a generic janitorial pouch. Same thing just purple.
	Key use time got reduced by a little less then 20% sounds like a lot but in testing it still seems pretty slow.
	
	Tested locally all seems to work.
	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -1022,7 +1022,7 @@ There are 9 wires.
 				playsound(loc, 'sound/items/keychainrattle.ogg', 30, 1, -2)
 			else
 				playsound(loc, 'sound/items/keychainrattle.ogg', 700, 1, -2)
-			if(do_after(user, 300, src)) //in ms so half a min of sitting their trying
+			if(do_after(user, 250, src)) //in ms so half a min of sitting their trying
```
Comment (Trilbyspaceclone) [inline code/game/machinery/doors/airlock.dm:1025]: `>:T`

File: code/game/jobs/job/civilian.dm [hunk 2]
```diff
@@ -178,11 +178,13 @@
 	health_modifier = 5
 	outfit_type = /decl/hierarchy/outfit/job/service/janitor
 
-	perks = list(PERK_MARKET_PROF, /datum/perk/job/jingle_jangle, /datum/perk/neat) //Union has revoked their chemistry privileges
+	perks = list(PERK_MARKET_PROF, /datum/perk/job/jingle_jangle, /datum/perk/neat, /datum/perk/sure_step) //Union has revoked their chemistry privileges
```
Comment (Trilbyspaceclone) [inline code/game/jobs/job/civilian.dm:181]: I dont think giving them a backround perk is a good idea for this

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm [hunk 3]
```diff
@@ -81,10 +81,12 @@
 	new /obj/item/clothing/shoes/galoshes(src)
 	new /obj/item/mop(src)
 	new /obj/item/soap/deluxe(src)
-	new /obj/item/storage/pouch/small_generic(src) // Because I feel like poor janitor gets it bad.
+	new /obj/item/storage/pouch/small_generic/purple(src) // These pouches never get used! Why not give them to are jani boys?
 	new /obj/item/newspaper_stack(src)
 	new /obj/item/holyvacuum(src)
 	new /obj/item/clothing/accessory/job/cape/service(src) // Boobles! - Seb
+	new /obj/item/tool/pickaxe(src) //helps with burrows! decent enough for minor pest control as well
+	new /obj/item/tool/hammer (src) //cracks! To go along side that pickaxe
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm:89]:
```suggestion
	new /obj/item/tool/hammer(src) //cracks! To go along side that pickaxe
```

File: code/game/jobs/job/civilian.dm (hunk 2 above)
Comment (benj8560) [reply code/game/jobs/job/civilian.dm:181]: Okay. Why?

File: code/game/machinery/doors/airlock.dm (hunk 1 above)
Comment (benj8560) [reply code/game/machinery/doors/airlock.dm:1025]: use words

File: code/game/jobs/job/civilian.dm (hunk 2 above)
Comment (benj8560) [reply code/game/jobs/job/civilian.dm:181]: does something break if you stack the perks or something? Or are you disagreeing from a balance perspective?

Comment (benj8560) [reply code/game/jobs/job/civilian.dm:181]: I'll just nope it anyways it was the only thing I was kinda iffy about in this PR felt abit to good.

Comment (benj8560) [reply code/game/jobs/job/civilian.dm]: please still explain if this is a issue with coding or balance. If its a code issue I'd like to understand what goes wrong

File: code/game/machinery/doors/airlock.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/game/machinery/doors/airlock.dm:1025]: me dislike `>:T`

This should not be nerfed as its easy to spam

Comment (benj8560) [reply code/game/machinery/doors/airlock.dm:1025]: but it's a buff. Unless I am misunderstanding how do_afters work

Comment (cdb-is-not-good) [reply code/game/machinery/doors/airlock.dm:1025]: She means nerfed as in 'the value shouldnt be reduced'. I disagree, personally-  many people have complained about the keys being useless even to janitors

Comment (benj8560) [reply code/game/machinery/doors/airlock.dm:1025]: I agree. My first step of the day as janitor is chucking those keys. It's not worth carrying them around. I'd say they are STILL not worth carrying but they are least not totally infuriating to use with this. By the time you find the key you could have just smashed the door, hacked it (not hard to afford gloves as a janitor) or just asked someone to come open it. This still gives plenty of time for the jingles to attracted someone.

Comment (benj8560) [reply code/game/machinery/doors/airlock.dm:1025]: Also as far as I'm aware these keys are mostly balanced around the Eris. We have a lot more doors and much bigger departments and the janitors not trying to clean one single mess they need into every single room to call it clean. It becomes a nightmare trying to use them for this.

Comment (du-top) [reply code/game/machinery/doors/airlock.dm:1025]: "Easy to spam"? How can "clicking once, tabbing out, cooking dinner, filing taxes, taking a nap, forgetting you're playing, clicking again, ..." be considered spamming? Yes, the keys are uselessly ... useless. A ridiculous solution to a non-problem. Janitor keeps stealing your stuff? Get them arrested and job-banned.
And don't get me started about that fucking keyring that's as big as the fucking vacuum.
Or the use of secure doors seemingly only "because they are blue'.
So in short, I agree with benj8560 and would buff it even more, thank you.

**Comment by cdb-is-not-good** [comment]
we can test this for now
